pcdsdevices/gon.py

```diff
@@ -655,6 +657,28 @@ def format_status_info(self, status_info):
 e_eta, e_chi, e_phi: {e_eta}, {e_chi}, {e_phi}
 x, y, z: {x}, {y}, {z} [{units}]
 """
+class HxrDiffractometer(BaseInterface, Device):
+    """
+    Class for diffractometer.
+
+    Parameters
+    ----------
+    name : str
+        A name to refer to the device
+
+    prefix_base : str
+        The EPICS base PV of the diffractometer stages
+    """
+
+    base_h = Cpt(BeckhoffAxis, 'BASE_H', kind='normal')
+    base_v = Cpt(BeckhoffAxis, 'BASE_V', kind='normal')
+    th = Cpt(BeckhoffAxis, 'CHI', kind='normal')
+    two_th = Cpt(BeckhoffAxis, 'TH', kind='normal')
+    chi = Cpt(BeckhoffAxis, '2TH', kind='normal')
```

Comment:
Fix variables names / PV suffix. th is mapping to CHI, etc

Let's call two_th - > tth instead